src/js/app.js

```diff
@@ -0,0 +1,95 @@
+var picArray = [];
+var count = 0;
+
+
+$(document).ready(function(){
+
+  $('#textInput').focus();
+
+	$('#formId').on('submit', function(e){
+
+		e.preventDefault();
+
+
+
+    word = $('#textInput').val();
+
+    $.get('http://api.giphy.com/v1/gifs/search', {
+
+      q: word,
+
+      api_key: 'dc6zaTOxFJmzC'
+
+    }).done(function(data){
+
+      var array = data.data;
+
+      for(i=0;i<array.length;i++){
+
+        var pic = array[i].images.original.url;
+
+        picArray.push(pic);
+
+       
+
+      }
+
+    }).fail(function(data){
+
+    }).always(function(data){
+
+    });
+
+    $('#textInput').val('');
+
+    $('#title').hide();
+    $('#textDiv').hide();
+    $('#buttonId').hide();
+    $('#stop').show();
+
+
+	});
+
+  $('#stop').on('click', function(e){
+
+    e.preventDefault();
+
+    location.reload();
```

Comment:
I'll probably mention this to everyone because it's a common theme, but `location.reload` requires another request from either the server or cache. It's actually faster to reset everything by using jQuery (which I noticed you started doing below)